source/clear-linux/tutorials/multi-boot/multi-boot-mint.rst

```diff
@@ -107,15 +103,15 @@ Install Mint\* 18.1 “Serena” MATE
 
       .. note:: The root partition UUID used below is unique to this example.
 
-      .. code-block:: console
+      .. code-block:: bash
 
-         # cd /boot/efi/loader/entries
+         sudo cd /boot/efi/loader/entries
 
-         # vi mint.conf
+         sudo vi mint.conf
```

Comment:
Consolidate lines 108 and 110 to "sudoedit sudo /boot/efi/loader/entries/mint.conf"

Reply:
Changed to "sudoedit /boot/efi/loader/entries/mint.conf"